python/konduit/base_inference.py

```diff
@@ -1155,10 +1155,10 @@ class ServingConfig(object):
     def __init__(
         self,
         http_port=None,
-        listen_host="localhost",
+        listen_host=None,
```

Comment:
sometimes it doesn't update right so might as well put `"localhost"` manually here